crates/swc_html_minifier/src/lib.rs

```diff
@@ -1533,17 +1533,25 @@ impl Minifier<'_> {
             },
         );
 
+        swc_ecma_visit::VisitMutWith::visit_mut_with(
```

Comment:
We should not apply this

Reply:
So it should only apply fixer but not `hygiene`? I would like to understand why.

Reply:
Done.